examples/seismic/tti/tti_example.py

```diff
@@ -7,7 +7,7 @@
 from examples.seismic.tti import AnisotropicWaveSolver
 
 
-def tti_setup(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
+def tti_setup(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=500.0,
```

Comment:
Don't like this bump to 500. See comment below

Reply:
`tn` was set again to be 250.0. It only is changed in the specific test as you suggested.